src/Stack/Git/GitOperations.cs

```diff
@@ -115,9 +122,9 @@ public string[] GetBranchesThatExistLocally(string[] branches)
 
     public string[] GetBranchesThatExistInRemote(string[] branches)
     {
-        var remoteBranches = ExecuteGitCommandAndReturnOutput($"ls-remote --heads origin {string.Join(" ", branches)}").Split(Environment.NewLine, StringSplitOptions.RemoveEmptyEntries);
+        var remoteBranches = ExecuteGitCommandAndReturnOutput("branch --remote --format=%(refname:short)").Split(Environment.NewLine, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Revert this as we now aren't using it for status and don't want to adjust the behaviour of other command just yet